apps/judicial-system/web/src/components/ProsecutorSelection/ProsecutorSelection.tsx

```diff
@@ -1,20 +1,18 @@
 import React, { useContext, useMemo } from 'react'
 import { useIntl } from 'react-intl'
-import { useQuery } from '@apollo/client'
 
 import { Select, Option } from '@island.is/island-ui/core'
 import { isIndictmentCase } from '@island.is/judicial-system/types'
 import {
   UserContext,
   FormContext,
 } from '@island.is/judicial-system-web/src/components'
-import { ProsecutorSelectionUsersQuery } from './prosecutorSelectionUsersGql'
-import {
-  User,
-  UserRole,
-} from '@island.is/judicial-system-web/src/graphql/schema'
+import { useProsecutorSelectionUsersQuery } from './prosecutorSelectionUsers.generated'
+
+import { UserRole } from '@island.is/judicial-system-web/src/graphql/schema'
 
 import { strings } from './ProsecutorSelection.strings'
+import { SingleValue } from 'react-select'
```

Comment:
Please order this import properly (at the top). 

I've created a PR that enforces this ordering via eslint so that we don't need to manually do this.

Reply:
Cool! I actually ended up just removing it as we don't need it